cli/tests/change_service_worker_migration.sh

```
@@ -0,0 +1,100 @@
+# But checking validity is good.
+# I'll include it.
+
+cd src/modules/my-service
+
+# Detect sed syntax (macOS vs Linux)
+if sed --version >/dev/null 2>&1; then
+  # GNU sed (Linux)
+  SED_INPLACE="sed -i"
+else
+  # BSD sed (macOS)
+  SED_INPLACE="sed -i ''"
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)